templates/shared/_security-fips.html

```diff
@@ -0,0 +1,115 @@
+<section class="p-strip">
+	<div class="row">
+		<div class="col-8">
+			<h1>Register your interest for the OpenJDK for FIPS beta</h1>
+			<p class="p-heading--four">In 2017, Canonical launched the Ubuntu FIPS programme to enable Linux workloads for US government regulated and high security environments, and now we are extending its coverage to Java workloads. This programme will give Ubuntu users free access to OpenJDK designed to comply with the FIPS requirements for beta testing for a 3 month period &ndash; please register your interest below to learn more.</p>
+		</div>
+  </div>
+	<div class="row">
+		<div class="col-8">
+			<form action="/marketo/submit" method="post" id="mktoForm_{{formid}}" >
+				<fieldset class="u-no-margin--bottom">
+					<h3 class="p-heading--five">Which Ubuntu LTS are you using?</h3>
+					<div class="row u-no-padding">
+						<div class="col-4 u-sv3">
+							<input type="checkbox" id="security-fips-xenial">
+							<label for="security-fips-xenial">16.04 (Xenial)</label>
+						</div>
+						<div class="col-4 u-sv3">
+							<input type="checkbox" id="security-fips-bionic">
+							<label for="security-fips-bionic">18.04 (Bionic)</label>
+						</div>
+						<div class="col-4 u-sv3">
+							<input type="checkbox" id="security-fips-focal">
+							<label for="security-fips-focal">20.04 (Focal)</label>
+						</div>
+					</div>
+				</fieldset>
+				<fieldset class="u-no-margin--bottom">
+					<h3 class="p-heading--five">Do you use any of the packages below?</h3>
+					<div class="row u-no-padding">
+						<div class="col-6">
+							<input type="checkbox" id="security-fips-tomcat">
+							<label for="security-fips-tomcat">tomcat</label>
+							<input type="checkbox" id="security-fips-other">
+							<label for="security-fips-other">Other</label>
```

Comment:
The other checkbox normally has an input field for the users to type. Take a look at https://ubuntu.com/openstack#get-in-touch for example